src/view/TerminalInterface.py

```diff
@@ -247,7 +247,23 @@ def getConfirmation(self, message, okStr="Y", nokStr="N"):
         okStr = okStr.lower()
         nokStr = nokStr.lower()
         self.__boldPrint(message + f" [{okStr}/{nokStr}]: ")
-        choice = str(self.__getUserInput())
+
+        while True:
+            if okStr == "scratch":
+                choice = str(self.__getUserInput())
+                break
+            elif keyboard.is_pressed("Y"):
+                choice = "y"
+                keyboard.press("backspace")
+                time.sleep(0.2)
+                print(" " + "y") 
+                break
+            elif keyboard.is_pressed("N"):
+                choice = "n"
+                keyboard.press("backspace")
+                print(" " + "n")
+                break

```

Comment:
Do we need an else for if the user wants to abort? Or am I thinking about a different place for that?